spec/service_consumers/pact_helper.rb

```diff
@@ -42,9 +42,15 @@
     stub_mvi unless example.metadata[:skip_mvi]
     stub_emis unless example.metadata[:skip_emis]
     stub_vet360 unless example.metadata[:skip_vet360]
+
+    Redis.current.flushall
```

Comment:
Were you run into issues with existing Redis keys when running the pact tests?

Reply:
yes the `there are no service history records` spec won't pass unless redis is cleared because the `at least one entry in the service history exists` redis data will still exist

Reply:
I ran into a similar issue with db records and transactional fixtures 👍